wof/wofpy_config.py

```diff
@@ -40,17 +38,13 @@
 _ODM2_TIMESERIES = os.path.join(_ROOT, 'examples', 'flask', 'odm2', 'timeseries')
 
 
-def makedirs(directory, overwrite='soft'):
-    if not os.path.exists(directory):
-        os.makedirs(directory)
-    elif os.path.exists(directory) and overwrite in ['soft', 'hard']:
-        if overwrite == 'hard':
-            print('Overwriting directory {}'.format(directory))
-            shutil.rmtree(directory)
-            os.makedirs(directory)
-    else:
-        msg = 'Directory {} already exists.'.format
-        raise ValueError(msg(directory))
+def makedirs(directory, overwrite=None):
+    if os.path.exists(directory) and overwrite == 'soft':
+        return
+    elif os.path.exists(directory) and overwrite == 'hard':
+        sys.stdout.write('Overwriting directory {}\n'.format(directory))
```

Comment:
Simpler logic to avoid bugs and use `stdout` instead of `print` to be more CLI "friendly."